### PR TITLE
VLO clean up: return an empty list instead of null

### PR DESCRIPTION
**Pull request by xnull**
## Overview

Description:
VLO clean up: return an empty list instead of null

Why should this be merged: 

Related issue(s) (if applicable): #<number>

## Checklist (Definition of Done):

- [ ] There are no TODOs left in the code
- [ ] [Coding conventions](https://github.com/CorfuDB/CorfuDB/wiki/Corfu-Style-Guidelines) (e.g. for logging, unit tests) have been followed
- [ ] Change is covered by automated tests
- [ ] Public API has Javadoc

### CODE REVIEW

File: runtime/src/main/java/org/corfudb/runtime/object/StreamViewSMRAdapter.java [hunk 1]
```diff
@@ -97,14 +97,13 @@ public void gc(long trimMark) {
...
-        if (data != null) {
-            if (data.getType() == DataType.DATA
-                    && data.getPayload(runtime) instanceof ISMRConsumable) {
-                return ((ISMRConsumable) data.getPayload(runtime))
-                        .getSMRUpdates(streamView.getId());
-            }
+        if (data == null
+                || data.getType() != DataType.DATA
+                || !(data.getPayload(runtime) instanceof ISMRConsumable)) {
+            return new ArrayList<>();
```
Comment (WenbinZhu) [inline runtime/src/main/java/org/corfudb/runtime/object/StreamViewSMRAdapter.java]: Can you use `Collections.emptyList()` instead?

File: runtime/src/main/java/org/corfudb/runtime/object/StreamViewSMRAdapter.java [hunk 2]
```diff
@@ -124,7 +123,8 @@ public void gc(long trimMark) {
             }
             data = streamView.previous();
         }
-        return null;
+
+        return new ArrayList<>();
```
Comment (WenbinZhu) [inline runtime/src/main/java/org/corfudb/runtime/object/StreamViewSMRAdapter.java]: Can you use `Collections.emptyList()` instead?

Comment (xnull) [reply runtime/src/main/java/org/corfudb/runtime/object/StreamViewSMRAdapter.java]: done

File: runtime/src/main/java/org/corfudb/runtime/object/StreamViewSMRAdapter.java (hunk 1 above)
Comment (xnull) [reply runtime/src/main/java/org/corfudb/runtime/object/StreamViewSMRAdapter.java]: done

**Pull request merged**